dotcom-rendering/playwright/lib/islands.ts

```diff
@@ -0,0 +1,25 @@
+import type { Page } from '@playwright/test';
+
+const waitForIsland = async (
+	page: Page,
+	island: string,
+	status: 'rendered' | 'hydrated' = 'rendered',
+	nth = 0,
```

Comment:
not important, but switching from individual parameters to object props here might be helpful in cases where we want to specify `nth` but just use the default `status` value.

Reply:
I really enjoyed reading the [Deno code style guide](https://docs.deno.com/runtime/manual/references/contributing/style_guide#exported-functions-max-2-args-put-the-rest-into-an-options-object) on function signatures.

Reply:
I agree and I did try this but I found the tests became really verbose. Actually what I could do is make `status` and `nth` part of an `options` param.